.gitignore

```diff
@@ -49,3 +49,6 @@ env/
 gamecenter/static/docs
 
 postgres
+
+# ignore the things inside of the log folder
+logs/*log.json
```

Comment:
Agreed. That will go in settings.py. When I tried it locally it was denied permission so I thought that could be something we could do later